addons/io/org.openhab.io.imperihome/README.md

```diff
@@ -7,12 +7,40 @@ It creates a REST service at _/imperihome/iss_ that implements the [ImperiHome S
 
 The ImperiHome integration service can be installed through the Paper UI. Navigate to Extensions &gt; Misc and click Install.
 
+If you want to use the latest snapshot or your own build of this binding, compile the code and copy the resulting JAR file into the _openhab/addons_ directory.
```

Comment:
This is nothing Imperi-Home add-on specific, so I would suggest to remove this here.